compact_str/src/traits.rs

```diff
@@ -71,6 +71,7 @@ unsafe impl LifetimeFree for Repr {}
 ///     * For floats we use [`ryu`] crate which sometimes provides different formatting than [`std`]
 impl<T: fmt::Display> ToCompactString for T {
     #[inline]
+    #[track_caller]
     fn to_compact_string(&self) -> CompactString {
```

Comment:
Perhaps it's a good idea adding a try_ version of this fn?

Reply:
Yes, it is! Added.